### PR TITLE
feat: use badge from UI Kit

### PR DESCRIPTION
**Pull request by weblancaster**
This is the work to remove the "custom" badge component and start using the `badge` component from the UI Kit.

**Checklist**
- [x] Did you add a JIRA issue in a commit message or as part of the branch name?
- [ ] Did you add new unit tests?
- [ ] Did you add new integration tests?
- [ ] If this is a regression, did you write a test to catch this in the future?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -76,6 +76,7 @@
     "@commitlint/cli": "6.1.3",
     "@commitlint/config-conventional": "6.1.3",
     "@dcos/eslint-config": "0.3.0",
+    "@dcos/ui-kit": "0.2.0",
```
Comment (weblancaster) [inline package.json]: Whenever https://github.com/dcos-labs/ui-kit/pull/21 is merged a new version of UI Kit needs to be released and then here I will need to bump to the newly released version.

**Comment by juliangieseke** [comment]
@weblancaster can you add a short descriptions why this is hold merge? Is it because of the `onClick` thing?

File: package.json (hunk 1 above)
Comment (Poltergeist) [reply package.json]: dcos-labs/ui-kit#66 is the update on this.

**Comment by weblancaster** [comment]
alfred test this

**Comment by weblancaster** [comment]
run integration tests

**Comment by weblancaster** [comment]
run integration tests

**Pull request merged**